https://gitlab.gnome.org/GNOME/gtk/merge_requests/67
https://bugs.gentoo.org/650536

Upstream seems to be unable to fix this throughly, so going with the rejected
fix until upstream finally gets their crap fixed for real.

--- gtk+-2.24.32/gtk/gtk.symbols
+++ gtk+-2.24.32/gtk/gtk.symbols
@@ -2498,8 +2498,8 @@
 #endif
 #endif
 
-#if IN_HEADER(__gtk_marshal_MARSHAL_H__)
-#if IN_FILE(__gtk_marshal_MARSHAL_C__)
+#if IN_HEADER(__gtk_marshal_MARSHAL_H__) || IN_HEADER(__GTK_MARSHAL_MARSHAL_H__)
+#if IN_FILE(__gtk_marshal_MARSHAL_C__) || IN_FILE(__GTK_MARSHAL_MARSHAL_C__)
 #ifndef GTK_DISABLE_DEPRECATED
 gtk_marshal_BOOLEAN__POINTER
 gtk_marshal_BOOLEAN__POINTER_INT_INT
